### ISSUE TITLE
Units do not reconnect if MQTT server is restarted

### ISSUE THREAD

**Issue by boy-wonder** [unixko/MitsuCon#17]
Thanks for this superb code which I use on all my units in my house. 

I use a stand alone Mosquito MQTT broker connected to Home Assistant. If I have to restart Mosquito, normally because I have to restart the hardware it is running on, MitsuCon does not reconnect. The only way to resolve this is to power off all the units and put them back on again. 

Perhaps there is a configuration issue I don't understand?

**Comment by unixko** [comment]
I use Mosquitto broker Add-ons in Hass.io without any restart problem. To separate problem please try:

- ping MitsuCon after restart server.
- check a broker log. It should have a log about client trying to reconnect but having problem. If no log at all make sure that your broker IP address doesn't change after restart.
- if you use Hass.io, use Mosquitto add-on with 1 MitsuCon to check that a problem is actually at MQTT broker or HA.

**Comment by boy-wonder** [comment]
Thanks for your help. I think I have resolved the problem. I did jump to conclusions I think.  It was HA configuration.

I have used HA for a long time and over that time many things have moved from manual yaml config to GUI based config using Integrations. I deleted my manual mqtt setup entries in configuration.yaml and recreated in the GUI and it seems to be OK now. Doesn't make sense to me, having read all the documentation, but there we go!
